pkg/arvo/app/lens.hoon

```diff
@@ -72,6 +91,27 @@
   :_  this(job.state ~)
   [bone.u.job.state %http-response (json-response:app (json-to-octs jon))]~
 ::
+++  diff-export
+  |=  [=wire data=*]
+  ^-  (quip move _this)
+  ::
+  ?>  ?=(^ job.state)
+  ::  herb will do whatever we tell it to, so by convention have it write to an
+  ::  app name based on the file name.
+  ::
+  ?>  ?=(%export -.source.com.u.job.state)
+  =/  app-name=tape  (trip app.source.com.u.job.state)
+  =/  output=@t  (crip "/{app-name}/jam")
+  ::
+  =/  jon=json
+    =/  =atom  (jam data)
+    =/  =octs  [(met 3 atom) atom]
+    =/  enc  (en:base64 octs)
+    (pairs:enjs:format file+s+output data+s+enc ~)
+  ::
+  :_  this(job.state ~)
+  [bone.u.job.state %http-response (json-response:app (json-to-octs jon))]~
```

Comment:
If we're only handling one `%diff`, I think we should make that explicit and send `[%pull /export]` or whatever at the end of this.